css/css-text/text-indent/text-indent-with-absolute-pos-child.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE HTML>
```

Comment:
Just for consistency:

```suggestion
<!DOCTYPE html>
```